js/__internal/grids/grid_core/columns_controller/m_columns_controller_utils.ts

```diff
@@ -563,7 +563,7 @@ export const updateColumnChanges = function (that: ColumnsController, changeType
   }
   if (columnIndex === undefined || columnIndex !== columnChanges.columnIndex) {
     if (isDefined(columnIndex)) {
-      columnChanges.columnIndices ??= [columnChanges.columnIndex];
+      columnChanges.columnIndices ??= isDefined(columnChanges.columnIndex) ? [columnChanges.columnIndex] : [];
```

Comment:
Let's just make a separate if statement, to make it a little more readable
```suggestion
      columnChanges.columnIndices ??= [];
      
      if(isDefined(columnChanges.columnIndex)) {
        columnChanges.push(columnChanges.columnIndex)
      }
```